server/templates/general/index.html

```diff
@@ -301,32 +130,42 @@
                 </div>
                 <div class="modal-content-block flex-container-center input-focus-within">
                     <span>Email</span>
-                    <input id="modal_user_email" type="email" value="" placeholder="" maxlength="20">
+                    <input id="modal_user_email" type="email" value="" placeholder="" maxlength="20" autocomplete="off">
                 </div>
                 <div class="modal-content-block flex-container-center input-focus-within">
                     <span>Password</span>
-                    <input  id="modal_user_password" type="password" value="" placeholder="" maxlength="20">
+                    <input  id="modal_user_password" type="password" value="" placeholder="" maxlength="20" autocomplete="new-password">
                 </div>
                 <div class="modal-content-block flex-container-center input-focus-within">
                     <span>Password</span>
-                    <input id="modal_user_confirm_password" type="password" value="" placeholder="" maxlength="20">
+                    <input id="modal_user_confirm_password" type="password" value="" placeholder="" maxlength="20" autocomplete="off">
+                </div>
+                <div class="modal-content-block flex-container-center input-focus-within">
+                    <span>Active</span>
+                    <div class="flex-container-center modal-container-checkbox">
+                        <div class="checkbox-container active-container-checkbox">
+                            <input id="modal_user_active" class="checkbox" type="checkbox" value="">
+                            <label for="checkbox">Active/Disabled</label>
+                        </div>
+                    </div>
                 </div>
                 <div class="modal-content-block flex-container-center input-focus-within">
                     <span>Roles</span>
-                    <div class="roles-container flex-container-center">
+                    <div class="modal-container-checkbox flex-container-center">
```

Comment:
As discussed earlier - we do not hard-code role checkboxes. Use rolesList variable to pull the full list of roles, acceptable by the server, and create a list of checkboxes based on that.